pkg/scheduler/event_handler.go

```diff
@@ -210,6 +210,8 @@ func (s *Scheduler) updateCluster(oldObj, newObj interface{}) {
 		// to the worker. Therefore, call Add func instead of Enqueue func.
 		s.clusterReconcileWorker.Add(oldCluster)
 		s.clusterReconcileWorker.Add(newCluster)
+	case !equality.Semantic.DeepEqual(oldCluster.Status.ResourceSummary, newCluster.Status.ResourceSummary):
+		s.clusterResourcesReconcileWorker.Enqueue(newCluster)
```

Comment:
My concern is that this will make the scheduler very sensitive because the cluster's resources will be updated frequently, when there are multiple clusters, it will put a lot of pressure on the scheduler.

What I've been thinking:
1. Is it reasonable for the scheduler to give up on scheduling a resource if it can not find a feasible cluster? Should the scheduler retry these tasks? such as by introducing a retry queue?
2. Karmada needs a capability that allows users to trigger rescheduling. There are many scenarios where rescheduling is required, cluster resource change is just one small case.